docs/preview/03-Features/powershell/azure-logic-apps-standard.md

```diff
@@ -0,0 +1,354 @@
+---
+title: "Azure Logic Apps - Standard"
+layout: default
+---
+
+# Azure Logic Apps - Standard
+
+## Installation
+
+To have access to the following features, you have to import the module:
+
+```powershell
+PS> Install-Module -Name Arcus.Scripting.LogicApps
+```
+
+## Cancel running instances for a Azure Logic App workflow
+
+Use this script to cancel all running instances for a specific Azure Logic App. 
+
+| Parameter                   | Mandatory | Description                                                                                                                                         |
+| --------------------------- | --------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `EnvironmentName`           | no        | The name of the Azure environment where the Azure Logic App resides. (default: `AzureCloud`)                                                        |
+| `ResourceGroupName`         | yes       | The resource group containing the Azure Logic App.                                                                                                  |
+| `LogicAppName`              | yes       | The name of the Azure Logic App containing the workflow for which the runs will be cancelled.                                                       |
+| `WorkflowName`              | yes       | The name of the workflow within the Azure Logic App for which the runs will be cancelled.                                                           |
```

Comment:
The `WorkflowName` was also optional, no?

Reply:
Ah, now I see it. This parameter is also responsible if the implementation is using consumption/standard, right?
Just a thought: this is a bit implicit, it might be a good idea to include `Standard`/`Consumption` in the function names? If yes: then we should make sure that we write an deprecated error when the 'old' ones, without the type, are still used.

Reply:
Added follow-up issue for this here: #419